### PR TITLE
updated README to refer to docs website for common queries

### PR DESCRIPTION
**Pull request by dunkOnIT**
The intention here is to reduce duplication of information (and thus risk of outdated information) as far as possible - so the README largely refers to the documentation website, which should be more actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,71 +2,21 @@
...
+- [API to read data from the website](https://wca-rest-api.robiningelbrecht.be/)
+- [Other WCA Repos](https://docs.worldcubeassociation.org/#wca-software-resources)
+- [Using OAuth, or writing data to the website](https://docs.worldcubeassociation.org/knowledge_base/v0_api.html)
+- [Joining WST]()
 
-We can run the application in two ways:
-- [Run using Docker](#run-using-docker)
-- [Run directly with Ruby](#run-directly-with-ruby-lightweight-also-only-runs-the-rails-portions-of-the-site)
+## Can I join WST?
+Currently (2023-09-10) WST is not accepting new members. We aim to onboard more volunteers by early 2024.
```
Comment (gregorbg) [inline README.md:17]: winter 2023 :muscle:

File: README.md [hunk 2]
```diff
@@ -2,71 +2,21 @@
...
-  cd worldcubeassociation.org
-  ```
+## Common Queries
+- [Overview of the WCA's software ecosystem](https://docs.worldcubeassociation.org/)
+- [Running the website locally](https://docs.worldcubeassociation.org/contributing/quickstart)
+- [Contributing Guide](https://docs.worldcubeassociation.org/contributing/detailed_contributing_guide.html)
+- [API to read data from the website](https://wca-rest-api.robiningelbrecht.be/)
+- [Other WCA Repos](https://docs.worldcubeassociation.org/#wca-software-resources)
+- [Using OAuth, or writing data to the website](https://docs.worldcubeassociation.org/knowledge_base/v0_api.html)
+- [Joining WST]()
```
Comment (gregorbg) [inline README.md:14]: empty link

File: README.md [hunk 3]
```diff
@@ -2,71 +2,21 @@
...
-- Clone this repo! (And navigate into it)
-  ```
-  git clone https://github.com/thewca/worldcubeassociation.org
-  cd worldcubeassociation.org
-  ```
+## Common Queries
+- [Overview of the WCA's software ecosystem](https://docs.worldcubeassociation.org/)
+- [Running the website locally](https://docs.worldcubeassociation.org/contributing/quickstart)
+- [Contributing Guide](https://docs.worldcubeassociation.org/contributing/detailed_contributing_guide.html)
+- [API to read data from the website](https://wca-rest-api.robiningelbrecht.be/)
```
Comment (gregorbg) [inline README.md:11]: This API is not officially endorsed by WST and should not appear on our front page readme

**Review by gregorbg — CHANGES_REQUESTED** [review]
Cool! Some nitpicks as usual.

In general, how confident are we that the links you're using here will stay stable? I want to avoid sending a PR to this repo's frontpage README every time we make a minute change to the docs

File: README.md (hunk 1 above)
Comment (dunkOnIT) [reply README.md]: Do you mean I should replace the date with winter 2023, or did you mean "winter 2024"? In either case, I am very opposed to specifying a season given the fact that we're having this conversation across hemispheres :P But I think specifying a quarter can help. I've made it Q2 2024 to give us some breathing room

File: README.md (hunk 3 above)
Comment (dunkOnIT) [reply README.md:11]: What would it take for the API to be endorsed by us? And what are the concerns with it not being endorsed?

I'm strongly in favour of linking to it for pragmatic reasons - it is our most common query, and enables hobbyist devs to quickly find what they're looking for.

Comment (dunkOnIT) [reply README.md:11]: We could change the wording to be more clear that it is unofficial?

Comment (gregorbg) [reply README.md:11]: The main concern is that people could think we are responsible for fixing errors, shielding feature requests or providing help when your app doesn't work

**Review by gregorbg — CHANGES_REQUESTED** [review]
Please change the wording of the Results API to make it **VERY** clear that it is not maintained by WST and we do not develop features for it nor do we provide any support for it.

File: README.md (hunk 1 above)
Comment (dunkOnIT) [reply README.md]: Reading this again, do you mean we'd look to onboard volunteers by the end of the year? Happy to change it to that if so

Comment (gregorbg) [reply README.md]: I think that we're in a position to onboard volunteers again around the year change 2023/2024

**Review by gregorbg — APPROVED** [review]

**Pull request merged**